src/borg/helpers.py

```diff
@@ -51,6 +51,22 @@ def Chunk(data, **meta):
     return _Chunk(meta, data)
 
 
+def get_rc():
+    return return_code
+
+
+def set_rc(rc):
+    global return_code
+    return_code = max(return_code, rc)
+    return return_code
+
+
+def init_rc():
+    global return_code
+    return_code = EXIT_SUCCESS
+    return return_code
```

Comment:
oh, oops. ok, keep it as is and maybe put a comment about that into that init fn.